types/src/front/lib/connectors_api.ts

```diff
@@ -304,6 +304,22 @@ export class ConnectorsAPI {
     return this._resultFromResponse(res);
   }
 
+  async getAllConnectorPermissions({
```

Comment:
I don't think we need to add this `getAllConnectorPermissions` - it would return too many nodes, and we just need the parents. The code should be ok and more consistent with the assistant config by fixing and using the existing `retrieveContentNodeParents`, called by `useParentResourcesById`. 
However this method does not correctly work in that case - only the nodes which are synchronized are returned (although it may depend on the implementation of the connector). For assistant, we display the tree starting on synchronized folder - all displayed folders are syncrhonized. For connector configuration, since we select roots, by definition parents are not syncrhonized, so no parents are retrurned. 
For google/microsft, if the node has no parentInternalId (so no synchronized parnet), it may be needed to fetch the item in google/ms, then get the parent from here.